src/user.rs

```diff
@@ -0,0 +1,46 @@
+use std::fs::File;
+use std::io::{Read, Write};
+use std::path::{Path, PathBuf};
+
+use rustc_serialize::json;
+
+use project_types::Project;
+use Error;
+
+
+pub fn new_user<P: AsRef<Path>>(project: String, public: P, name: String) -> Result<(), Error> {
```

Comment:
You're right that is a problem.

1. Project and User should definitely hold `String` not `&str`.
2. Args contains a String, and we move that String out of Args during unwrap.

Yeah you're right, all the params to new_user should be String.